### PR TITLE
ddt removal in cli content view filter tests

### PR DESCRIPTION
**Pull request by sthirugn**
This PR also:
* Made pylint 100%
* optimized the tests to remove redundant setup calls to improve test execution time

### CODE REVIEW

**Comment by sthirugn** [comment]
Test results look a lot better than latest jenkins results (which shows 13 minutes)
```sh
Finding files... done.
Importing test modules ... done.

----------------------------------------------------------------------
Ran 29 tests in 319.469s

OK (skipped=2)
```

**Comment by elyezer** [comment]
Looks good, ACK.

File: tests/foreman/cli/test_contentviewfilter.py [hunk 1]
```diff
@@ -60,25 +48,30 @@ def test_create_cvf_with_different_names(self, name):
...
+                    'content-view-id': self.content_view['id'],
+                    'type': filter_content_type,
+                    'name': name,
+                })
+                self.assertEqual(result.return_code, 0)
+                self.assertEqual(len(result.stderr), 0)
+
+                result = ContentView.filter_info({
+                    u'content-view-id': self.content_view['id'],
+                    u'name': name
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_contentviewfilter.py]: a few more of these missing commas throughout the file, but no biggie

Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: ah yeah, let me fix it quickly

**Comment by JacobCallahan** [comment]
ACK, pending minor comment

**Comment by sthirugn** [comment]
@JacobCallahan I now fixed more than 10 instances of missing commas in dictionaries. I hope I fixed them all :)

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**